### PR TITLE
[Beta-0.3.0] Adding Presets functionality

### PR DESCRIPTION
**Pull request by buffet-time**
Site Beta-0.3.0

### Major changes

- Presets!
  -  Added functionality to handle prefedined presets
  - Top 42 and Top 100 are the first 2
    - Planned functionality for user to create custom Presets
- Added modals throughout the site to support more complex UX interactions
- Various bug fixes and style changes

### CODE REVIEW

**Review by APeninsula — APPROVED** [review]
Works on my end, no issues with custom sizeable charts either.

**Review by zysim — APPROVED** [review]
Looks good

**Review by erunks — APPROVED** [review]
Looks good AFAIK 👍

File: src/components/Sidebar.vue [hunk 1]
```diff
@@ -2,44 +2,117 @@
...
+		<dialog ref="siteInfo" class="bg-transparent">
+			<div
+				class="bg-neutral-700 p-5 flex flex-col gap-2 justify-center items-center text-white"
+			>
+				<!-- Header -->
+				<p class="text-neutral-200">Site Beta-{{ version }}</p>
+
+				<p>This site will NEVER have Ads or any form of tracking</p>
+				<p>
+					Disocrd is the preferred platform for discussion and bug reports of
```
Comment (niiree) [inline src/components/Sidebar.vue:52]: woops!

File: src/App.vue [hunk 2]
```diff
@@ -12,38 +12,16 @@ GlobalSiteOptions.value = GetSiteOptions()
...
-			1) default options for top 10, top 25, top 50, top 100
-			2) have some sort of ui to create these yourself
-	-->
-
 	<!-- in case the OG site goes down a reference image -->
 	<!-- https://i.gyazo.com/9fb8f19495588fd3b31c47389d248bcb.png -->
-	<div class="flex">
-		<Sidebar class="tw-sidebar-width" @can-render-chart="renderChart = true" />
-		<Chart v-if="renderChart" class="w-4/5" />
+	<div class="flex text-white text-center font-sans antialiased">
```
Comment (niiree) [inline src/App.vue:17]: Isn't `antialiased` (that is, grayscale + regular antialias options for osx/win respectfully) a default option in most (if not all) browsers? Why the enforcement?

Just curious overall.

File: src/components/Sidebar.vue (hunk 1 above)
Comment (buffet-time) [reply src/components/Sidebar.vue:52]: Lol developer minded people of course can use GitHub but I'd prefer non technical people from posting here :P

**Pull request merged**